rails/react-components/src/library/components/permission-forms-v2/common/types.ts

```diff
@@ -1,9 +1,10 @@
+export type CurrentSelectedProject = number | null;
 export interface IPermissionForm {
   id: string;
   name: string;
   is_archived: boolean;
   can_delete: boolean;
-  project_id?: number | string; // need to fix this
+  project_id: CurrentSelectedProject;
```

Comment:
I think this should be `project_id: number` since the Portal API always returns a number. I guess it won't break anything, but it's better for the sake of documenting the API output.

Reply:
ah, thanks, I see.